lib/command/src/Obelisk/Command/Thunk.hs

```diff
@@ -621,6 +626,47 @@ packThunk' noTrail thunkDir = checkThunkDirectory thunkDir >> readThunk thunkDir
       liftIO $ createThunk thunkDir thunkPtr
       pure thunkPtr
 
+-- set thunk to a desired branch
+setThunk :: MonadObelisk m => String -> FilePath -> m ()
+setThunk branch thunkDir = checkThunkDirectory thunkDir >> readThunk thunkDir >>= \case
+  Left err -> failWith $ T.pack $ "thunk set : " <> show err
+  Right (ThunkData_Packed thunkptr) -> do
+    let thunkSrc = _thunkPtr_source thunkptr
+        repository = case thunkSrc of
+          ThunkSource_Git gitSource -> T.unpack $ render $ _gitSource_url gitSource
+          ThunkSource_GitHub gitHubSource -> T.unpack $ render $ _gitSource_url $ forgetGithub True gitHubSource
+    -- check to see if branch exists before modifying directory
+    (exitCode, _) <- gitLsRemoteExitCode repository branch
```

Comment:
The unused value is different than `remoteResults`?

Reply:
The unused value is the `GitRef` type which seems to give me a ref and not an actual sha1 rev.